samples/ipc_telemetry/common/sensors_1_model.h

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/********************************************************************
+ * This code was auto-generated from sensors.1 DL and shall not be
```

Comment:
should the autogenerated files be placed in an _internal/ directory and have the underscore prefix as well? i.e. "_sensors_1_model.h" to show users should not touch these kinds of files?

Reply:
I don't think so, it would send the message that developers should not use the content of this file, which is not the case. We do not have any special format for autogenerated code in our guidelines.